capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.module

```diff
@@ -293,3 +293,19 @@ function c4m_content_group_submit_redirect(&$form, &$form_state) {
   // Redirect a user to homepage.
   $form_state['redirect'] = url('', array('absolute' => TRUE));
 }
+
+/**
+ * Implements hook_custom_theme().
+ *
+ * Set Kapablo theme for group creation page.
+ */
+function c4m_content_group_menu_alter(&$items) {
```

Comment:
This is not custom_theme, Could we do it like:
```
/**
 * Implements hook_custom_theme
 */
function MYMODULE_custom_theme() {
  // Set path specific theme.
  if (current_path() == 'node/add/group') {
    return variable_get('custom_theme', 'theme_name');
  }
}
```

@amitaibu Which one is better ?